debian/changelog

```diff
@@ -1,3 +1,17 @@
+python-securesystemslib (0.18.0-1) unstable; urgency=medium
+
+  * New upstream release includes among other things:
+    - an enhanced key generation, export and import interface,
+    - updated documentation of interface functions,
+    - and more and better tests.
+
+  * Update d/control to:
+    - remove obsolete python-dateutil dependency,
+    - bump standards version to 4.5.1,
+    - and compat to 13.
```

Comment:
Nit:
```suggestion
    - increase compat to 13.
```
or bump?

Reply:
Oh, I only saw this just now. I had conceived this as a sentence, that is ...
```
 Update d/control to remove obsolete python-dateutil dependency, bump standards version to 4.5.1, and compat to 13.
```
But then I'd maybe have to change the punctuation? 

Anyways, given that 0.18.0-1 is already released, I I'll make use of my non-native-speaker bonus and leave this as is. :P

But thanks for the thorough review, @joshuagl!